### PR TITLE
[SCIP] patch bump

### PR DESCRIPTION
**Pull request by matbesancon**
Patch bump which will hopefully fix windows

### CODE REVIEW

**Comment by matbesancon** [comment]
ping @rschwarz hopefully this could fix the windows build issues

**Comment by rschwarz** [comment]
Worth a try :crossed_fingers:

File: S/SCIP/build_tarballs.jl [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
 sources = [
-    ArchiveSource("https://scip.zib.de/download/release/scipoptsuite-7.0.1.tgz", "971962f2d896b0c8b8fa554c18afd2b5037092685735d9494a05dc16d56ad422")
+    ArchiveSource("https://scip.zib.de/download/release/scipoptsuite-7.0.2.tgz", "f81b5a2c1c0eb949cf06bd50f42826e55284fa1269a6f28a92ac1a06d9c93a03")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir
-cd scipoptsuite-7.0.1/
+cd scipoptsuite-7.0.2/
```
Comment (giordano) [inline S/SCIP/build_tarballs.jl:16]:
```suggestion
cd scipoptsuite*
```

File: S/SCIP/build_tarballs.jl [hunk 2]
```diff
@@ -48,9 +48,9 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="bliss_jll", uuid="508c9074-7a14-5c94-9582-3d4bc1871065"))
-    Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d"), v"6.1.2")
-    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
+    Dependency(PackageSpec(name="bliss_jll", uuid="508c9074-7a14-5c94-9582-3d4bc1871065")),
+    Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d", version=v"6.1.2")),
```
Comment (giordano) [inline S/SCIP/build_tarballs.jl:52]: Ok, I just merged but I don't understand this

Comment (matbesancon) [reply S/SCIP/build_tarballs.jl:52]: I hadn't noticed the reordering. I had an error in the constructor Dependency(PkgSpec, version), so I integrated the version requirement in the PkgSpec Constructor

Comment (giordano) [reply S/SCIP/build_tarballs.jl:52]: Well, if you didn't do that on purpose I believe that was wrong

Comment (giordano) [reply S/SCIP/build_tarballs.jl:52]: > I had an error in the constructor Dependency(PkgSpec, version)

Did you have it locally?

Comment (matbesancon) [reply S/SCIP/build_tarballs.jl:52]: Yes:
```
julia> dependencies = [
           Dependency(PackageSpec(name="bliss_jll", uuid="508c9074-7a14-5c94-9582-3d4bc1871065"))
           Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d"), v"6.1.2")
           Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
       ]
ERROR: MethodError: no method matching Dependency(::Pkg.Types.PackageSpec, ::VersionNumber)
Closest candidates are:
  Dependency(::Pkg.Types.PackageSpec) at /home/mbesancon/.julia/packages/BinaryBuilderBase/fylAz/src/Dependencies.jl:29
  Dependency(::Any) at /home/mbesancon/.julia/packages/BinaryBuilderBase/fylAz/src/Dependencies.jl:29
Stacktrace:
 [1] top-level scope at REPL[12]:1

julia> dependencies = [
           Dependency(PackageSpec(name="bliss_jll", uuid="508c9074-7a14-5c94-9582-3d4bc1871065"))
           Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d", version=v"6.1.2"))
           Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
       ]
3-element Array{Dependency,1}:
 Dependency(PackageSpec(
  name = bliss_jll
  uuid = 508c9074-7a14-5c94-9582-3d4bc1871065
  version = *
))
 Dependency(PackageSpec(
  name = GMP_jll
  uuid = 781609d7-10c4-51f6-84f2-b8444358ff6d
  version = VersionSpec("6.1.2")
))
 Dependency(PackageSpec(
  name = CompilerSupportLibraries_jll
  uuid = e66e0078-7015-5450-92f7-15fbd957f2ae
  version = *
))
```

Comment (giordano) [reply S/SCIP/build_tarballs.jl:52]: Julia v1.5 I guess?

Comment (giordano) [reply S/SCIP/build_tarballs.jl:52]: Anyway, this line is wrong, need to bring it back as it was before

Comment (matbesancon) [reply S/SCIP/build_tarballs.jl:52]: > Julia v1.5 I guess?

Yes, I am not part of the cool kis running on a bleeding master

**Pull request merged**